model-api/bpmn-model/src/test/java/org/operaton/bpm/model/bpmn/OperatonExtensionsTest.java

```diff
@@ -97,17 +102,17 @@ void testAssignee(String namespace, BpmnModelInstance modelInstance) {
   @ParameterizedTest(name = "Namespace: {0}")
   void testAsync(String namespace, BpmnModelInstance modelInstance) {
     initOperatonExtensionsTest(namespace, modelInstance);
-    assertThat(startEvent.isOperatonAsync()).isFalse();
-    assertThat(userTask.isOperatonAsync()).isTrue();
-    assertThat(parallelGateway.isOperatonAsync()).isTrue();
+    assertThat(startEvent.isOperatonAsyncBefore()).isTrue();
```

Comment:
**Review Hint**: Changed expectation to true for startEvent, since the test resource is defined so.

<img width="465" alt="Screenshot 347" src="https://github.com/user-attachments/assets/c563a3f5-1d6c-4f1a-9c23-575e7f21bbd9" />